unraid_mcp/tools/storage.py

```
@@ -274,4 +274,56 @@ def format_bytes(bytes_value: int | None) -> str:
+              total
+            }
+          }
+        }
+        """
+        try:
+            logger.info("Executing delete_archived_notifications tool")
+            response_data = await make_graphql_request(query)
+            return response_data.get("deleteArchivedNotifications", {})
+        except Exception as e:
+            logger.error(f"Error in delete_archived_notifications: {e}", exc_info=True)
+            raise ToolError(f"Failed to delete archived notifications: {str(e)}") from e
```

Ruff (0.15.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)